include/gz/sim/components/Gravity.hh

```diff
@@ -41,6 +41,10 @@ namespace components
   using GravityEnabled = Component<bool, class GravityEnabledTag>;
   GZ_SIM_REGISTER_COMPONENT(
       "gz_sim_components.GravityEnabled", GravityEnabled)
+
+      /// \brief Store the gravity acceleration.
```

Comment:
Nit: Update indentation. Also, is the comment correct? I'd expect something about store whether we want to enable gravity or not.